### PR TITLE
Add few simply reply

### PR DESCRIPTION
**Pull request by Nofated095**
- &dns
- &launcher
- &pw
- &log.minecraft
- &log.launcher

### CODE REVIEW

File: texts.py [hunk 1]
```diff
@@ -33,3 +33,12 @@ class TextFields(object):
     pay_for_help = '在群里和大佬吹牛逼帮助不了你的问题？\nhttps://afdian.net/@tnqzh123 \n购买一对一帮助服务即可快速解决你的问题！'
     question_keywords: List[str] = ['为什么', '怎么回事', '为啥', '问个问题', '请问', '问一下', '如何解决',
                                     '我想问', '什么问题', '咋回事', '怎么办', '怎么解决']
+    dns = '我们认为可能你的 DNS 设置导致了一系列问题，您可以参考 http://dwz.win/arh5 的内容并将 DNS 设置为 223.5.5.5 和 8.8.8.8'
+    launcher = '''一个问题在启动器里面反复横跳？也许你可以尝试更换启动器！以下是几个推荐的启动器
+HMCL: https://ci.huangyuhui.net/job/HMCL/
+Baxa: https://www.bakaxl.com/
+PCL2: https://afdian.net/@LTCat?tab=feed'''
+    log_minecraft = '我们需要根据你的游戏日志对问题进行判断，一般日志文件存放在 .minecraft/versions/log/latest.log'
```
Comment (SerinaNya) [inline texts.py]: 请使用启动器的「测试游戏」功能启动游戏，并在复现问题后导出日志发送至群内。如果问题与外置登录有关，请在启动器的「 JVM 参数」设置中填入 -Dauthlibinjector.debug

File: texts.py [hunk 2]
```diff
@@ -33,3 +33,12 @@ class TextFields(object):
...
+    log_launcher = '我们需要根据你的启动器日志对问题进行判断，一般情况下你可以在设置中导出启动器日志'
```
Comment (SerinaNya) [inline texts.py]: 请在启动器中复现你的问题，然后导出启动器日志发送至群内

**Review by SerinaNya — CHANGES_REQUESTED** [review]
dns launcher email change_password 不要  
那两个改一下

**Comment by Mashirl** [comment]
ping @jinzhijie

**Comment by Mashirl** [comment]
Maybe we need a "&log" command to send both messages. @jinzhijie

**Review by SerinaNya — APPROVED** [review]

**Comment by SerinaNya** [comment]
Why we need it? What should it do?

**Comment by Qubik65536** [comment]
> Why we need it? What should it do?

确实有些人不知道怎么找到log

**Comment by Nofated095** [comment]
Thanks

**Pull request merged**